### DISCUSSION TITLE
Centralised optimisation of heating power demand

### DISCUSSION THREAD

**Discussion by pbartolo26** [jmcollin78/versatile_thermostat#640] [Ideas]
To optimise the use of the energy produced by photovoltaic panels as much as possible, the idea would be to run the heating systems sequentially to avoid a power demand greater than solar production capacity. This must be done in accordance with the lighting percentage instructions calculated by the thermostat.

**Comment by jmcollin78** [comment]
Hello @pbartolo26 ,

Like this for example ? 😉 

<img width="858" alt="Capture d’écran 2024-11-15 à 23 21 26" src="https://github.com/user-attachments/assets/e904fcd5-2326-4d08-b3bc-cad10c1294ea">

If yes, it is already done.

**Comment by pbartolo26** [comment]
Bonjour Jean-Marc,

Je me permets de t'écrire en français car je devine que tu es francophone
vu ton prénom ;)



Je n'étais pas au courant de ce dispositif sur le thermostat. En tout cas
je ne comprends pas bien comment l'algorithme fonctionne car chez moi ça
n'a pas l'air de fonctionner très bien  :
[image: image.png]

Les périodes d'activation sont différentes sur mes 2 chauffages car, mais
c'est peut-être une erreur, ayant un plancher chauffant à base de
résistances, j'ai pensé qu'une période de 30 min était plus adaptée à un
chauffage à inertie. En tout cas on constate bien une activation simultané
des 2 chauffages.

Sachant que j'ai des panneaux solaires qui font 3 kilowatts crête, mon
plancher chauffant 4 kilowatts virgule et mon chauffage supplémentaires un
kilowatt, cela pourrait être un peu mieux optimisé.

Je suis preneur de tes conseils  !

Bien cordialement.

Patrick

**Reply by jmcollin78** [reply]
Hello @pbartolo26 ,

Oui, je suis Français, mais on est aussi lu par des non Fr et comme c'est souvent les mêmes questions ça peut les aider aussi.

Sinon, on ne voit pas ton image :
![Capture d’écran 2024-11-18 à 10 45 33](https://github.com/user-attachments/assets/ec84c3cf-d37d-4b24-bcdf-3e48d158f7f4)

et donnes moi les attributs de ton VTherm (Outils de dev / Etats) sous cette forme : https://github.com/jmcollin78/versatile_thermostat/blob/main/README.md#custom-attributes
